groups/templates/groups/group_edit.html

```diff
@@ -1,4 +1,7 @@
 {% include 'base.html' %}
+{% block content %}
+{% load tz %}
+{% timezone "US/Pacific" %}
```

Comment:
and there should be no hardcoded default time zone in the code (e.g. use the one from the server by default).